doc/users/project/mission.rst

```diff
@@ -0,0 +1,29 @@
+Matplotlib Mission Statement
+============================
+
+The Matplotlib developer community develops, maintains, and supports Matplotlib
+and its extensions to provide plotting tools for the Scientific Python
+Ecosystem.  Our success is based on how effectively we enable our users, across
+scientific, industrial, commercial, and educational domains, to achieve their
+goals.
+
+We believe that a diverse developer community creates the best software, and we
+welcome anyone who shares our mission and values, as described in the code of
+conduct.
+
+
+Adapting the requirements :ref:`laid out by John Hunter
+<History>` Matplotlib should:
+
+* Support all users of the Scientific Python Ecosystem;
+* Produce high-quality raster and vector formats output;
+* Be embeddedable in graphical user interfaces for application development,
+  standalone, and on the web.
```

Comment:
"standalone" seems logically orphaned between two embeddings?  Maybe:
```suggestion
* Be usable as a stand-alone graphical user interface, and to be embeddable in applications, both native and on the web.
```

Reply:
I agree this is a better wording.